lib/proportions.js

```diff
@@ -107,16 +109,34 @@ define(['d3-interpolate', 'virtual-dom', 'filters/genericnode', 'helper'],
           return _.t('node.deactivated');
         });
 
+        nodes.forEach(function (node) {
+          if (node.flags.gateway) {
+            var mesh = node.nodeinfo.network.mesh;
+            mesh[Object.keys(mesh)[0]].interfaces.tunnel.forEach(function (mac) {
```

Comment:
static config vs this solution. some communities use mzbat or bat0 or... Only way to select the first object.